### ISSUE TITLE
Request to support for -- parameter separator.

### ISSUE THREAD

**Issue by lygstate** [tj/commander.js#374]

**Comment by SomeKittens** [comment]
Can you provide some examples, as well as why this works better for you than other current solutions?

**Comment by SomeKittens** [comment]
Closing, will reopen if examples provided.

**Comment by ejfrancis** [comment]
if @lygstate was talking about the ability to add a separator between options, I've done that in my fork in the `feature/option-separator` branch https://github.com/ejfrancis/commander.js/commit/1adcfceb6d8e16e1298eaaea77709a4304e29684

So you can do

```
  program
    .optionSeparator('Section A')
    .option('-a, --aopt', ' aopt desc')
    .option('-b, --bopt', ' bopt desc')
    .optionSeparator('Section B')
    .option('-c, --copt', ' copt desc')
    .option('-d, --dopt', ' dopt desc')
```

and its output looks like this

![image](https://cloud.githubusercontent.com/assets/3171352/15408455/1efeec80-1dc5-11e6-8559-1397288611d2.png)


@tj is this something you're interested it getting into the repo? if so I could test it and put up a PR

**Comment by bimlas** [comment]
I think @lygstate thought the way the Git works. For example:

```
git log [<options>] [<revision range>] [[--] <path>…​]
```

Thus, it would be possible to use multiple variadic arguments.

```
mystuff <command> [tags...] -- [paths...]
```

**Comment by jkowalleck** [comment]
> I think [@lygstate](https://github.com/lygstate) thought the way the Git works. For example:
> 
> ```
> git log [<options>] [<revision range>] [[--] <path>…​]
> ```
> 
> Thus, it would be possible to use multiple variadic arguments.
> 
> ```
> mystuff <command> [tags...] -- [paths...]
> ```

that's a feature i'd also like to see.
has this been added to v14? how to enable it?

**Comment by shadowspawn** [comment]
There is not support for multiple variadic command-arguments.

#2328 is about grouping together options/commands in the help under a different heading than Options/Commands.

**Comment by jkowalleck** [comment]
> There is not support for multiple variadic command-arguments.
> 
> [#2328](https://github.com/tj/commander.js/pull/2328) is about grouping together options/commands in the help under a different heading than Options/Commands.

Exactly. Then why does https://github.com/tj/commander.js/pull/2328 mention this very issue if they dont have anything in common? 
This gave me the impression that this thing would have been implemented in the #2328 -- confusing.

**Comment by shadowspawn** [comment]
Ah. The comment on this issue by @ejfrancis is about help groups, and hence the link from #2328 to the comment (and #2328 not relevant to the rest of this issue).

https://github.com/tj/commander.js/issues/374#issuecomment-220442296
